<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SysTray-X Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SysTray-X.qrc">
    <normaloff>:/files/icons/SysTray-X.png</normaloff>:/files/icons/SysTray-X.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="defaultIconGroupBox">
       <property name="title">
        <string>Default icon</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QGroupBox" name="iconGroupBox">
          <property name="title">
           <string>Icon</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QRadioButton" name="defaultIconRadioButton">
               <property name="text">
                <string>Default Thunderbird</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">defaultIconTypeGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="lookThroughDefaultIconRadioButton">
               <property name="text">
                <string>Lookthrough icon</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">defaultIconTypeGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QRadioButton" name="customDefaultIconRadioButton">
                 <property name="text">
                  <string>Custom icon</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">defaultIconTypeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="defaultImageLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="chooseCustomDefaultIconButton">
                 <property name="text">
                  <string>Choose</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hideDefaultIconCheckBox">
          <property name="text">
           <string>KDE integration (Hide)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="iconTypeGroupBox">
       <property name="title">
        <string>Mail notification icon</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QRadioButton" name="blankRadioButton">
            <property name="text">
             <string>Blank icon</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">iconTypeGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QRadioButton" name="newMailButton">
            <property name="text">
             <string>New mail icon</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">iconTypeGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QRadioButton" name="customRadioButton">
            <property name="text">
             <string>Custom icon</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">iconTypeGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="imageLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chooseCustomButton">
            <property name="text">
             <string>Choose</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QRadioButton" name="noIconRadioButton">
            <property name="text">
             <string>No icon</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">iconTypeGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QRadioButton" name="tbIconRadioButton">
            <property name="text">
             <string>Default Thunderbird</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">iconTypeGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="themeGroupBox">
       <property name="title">
        <string>Theme</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QRadioButton" name="lightRadioButton">
            <property name="text">
             <string>Light</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">themeGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="darkRadioButton">
            <property name="text">
             <string>Dark</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">themeGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Windows</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Minimize</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QRadioButton" name="defaultMinimizeRadioButton">
                 <property name="text">
                  <string>Default minimize</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">minimizeTypeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="minimizeMethod1RadioButton">
                 <property name="text">
                  <string>Minimize to tray</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">minimizeTypeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="minimizeMethod2RadioButton">
                 <property name="text">
                  <string>Minimize to tray, method 2</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">minimizeTypeGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Close</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QRadioButton" name="defaultCloseWindowsRadioButton">
                 <property name="text">
                  <string>Default close</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">closeTypeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="minimizeMainCloseChildrenWindowsRadioButton">
                 <property name="text">
                  <string>Minimize main, close children</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">closeTypeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="minimizeAllWindowsRadioButton">
                 <property name="text">
                  <string>Minimize all windows</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">closeTypeGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="startMinimizedCheckBox">
              <property name="text">
               <string>Start application minimized</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="restorePositionscheckBox">
              <property name="text">
               <string>Restore startup positions</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Number properties</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="showNumberCheckBox">
            <property name="text">
             <string>Display unread message count</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Number color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="numberColorPushButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Number size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numberSizeSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="numberAlignmentLabel">
              <property name="text">
               <string>Alignment</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="numberAlignmentComboBox">
              <property name="currentIndex">
               <number>4</number>
              </property>
              <item>
               <property name="text">
                <string>Top left</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Top centre</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Top right</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Middle left</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Middle centre</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Middle right</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bottom left</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bottom centre</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bottom right</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="numberMarginsLabel">
            <property name="text">
             <string>Margins (left, top, right, bottom):</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QSpinBox" name="numberMarginLeftSpinBox">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numberMarginTopSpinBox">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numberMarginRightSpinBox">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numberMarginBottomSpinBox">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="countTypeGroupBox">
            <property name="title">
             <string>Count type</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QRadioButton" name="unreadRadioButton">
                 <property name="text">
                  <string>Unread</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">countTypeGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="newRadioButton">
                 <property name="text">
                  <string>New</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">countTypeGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="debugWindowCheckBox">
       <property name="text">
        <string>Display debug window</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SysTray-X.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="minimizeTypeGroup"/>
  <buttongroup name="defaultIconTypeGroup"/>
  <buttongroup name="closeTypeGroup"/>
  <buttongroup name="themeGroup"/>
  <buttongroup name="countTypeGroup"/>
  <buttongroup name="iconTypeGroup"/>
 </buttongroups>
</ui>
